src/main.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 
 import App from './App.tsx';
 
 import './index.css';
 
 ReactDOM.createRoot(document.querySelector('#root')!).render(
     <React.StrictMode>
-        <App />
+        <BrowserRouter basename="/fe-react-2024">
```

Comment:
try to change this. Use hash router, because now navigation isn't working for deployed version and I can't check your homework.